### PR TITLE
[SELS-TASK] - [BE] - Save the users answer in answer table

### PR DESCRIPTION
**Pull request by Mjdelima20**
###  [BE] - Save the users answer in answer table

**[Commands]**

- `docker-compose up`

**[Pre-conditions]**
- Open Postman and input the following endpoints
- **For submiting the answer** 
   -  _req: POST_
   - `/api/student/category/{category_id}/word/{word_id}/answer`
   - `JSON`
        - {
  "answer": "the User answer",
  "question_no": 22
}
 
image

image

### CODE REVIEW

File: backend/src/categories/words.ts [hunk 1]
```diff
@@ -7,25 +7,32 @@ import {
...
+import { Choices } from './choices';
+import { Answers } from '../answer/answers';
 
 @Entity('words')
 export class Words {
   @PrimaryGeneratedColumn()
   word_id: number;
 
-  @Column({ nullable: false })
+  @Column({ default: 1 })
```
Comment (RDLegs) [inline backend/src/categories/words.ts:18]: Revert this change

**Review by RDLegaspi — APPROVED** [review]
LGTM

**Review by RDLegs — APPROVED** [review]
LGTM!
